### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by Dentrax** [vmware-tanzu/k-bench#36]
_TL;DR: I'm wondering if this project is still supported and maintained._

---

We recently used k-bench to run some benchmark test against kubernetes clusters, but we are now a little worried that the project might be unmaintained since no PRs have been merged or issue discussion for a long time. And there has not been a new release. It seems maintainers no longer active on GitHub too: @ganesank-git @yonglipa @ganesank

I think looking for new maintainers to help out with the workload would help. This is a pretty cool project which I hope will continue to improve! And we'd like to maintain it!

**Comment by yonglipa** [comment]
Hi Furkan,

Thanks for reaching out and showing interest in the k-bench project. Yeah,
as you mentioned this is no longer actively maintained. I am cc’ing
Karthik, who may know better update-to-date context and plan for this
project. Having said that, if you would like to become a maintainer of this
project, it is more than welcomed and we could add your GitHub account to
the repository. Please let me know,

Best regards,

yong

**Comment by yonglipa** [comment]
Hi Furkan,

I will soon be publishing a roadmap and will start investigating some of the issues that have been raised. Appreciate your interest.
We welcome your feedback and contributions.

Thanks,
Karthik

From: Yong Li ***@***.***>
Date: Tuesday, November 2, 2021 at 10:53 AM
To: Karthik Ganesan ***@***.***>, vmware-tanzu/k-bench ***@***.***>
Cc: Mention ***@***.***>, vmware-tanzu/k-bench ***@***.***>
Subject: Re: [vmware-tanzu/k-bench] Is this project still maintained? (Issue #36)

Hi Furkan,

Thanks for reaching out and showing interest in the k-bench project. Yeah, as you mentioned this is no longer actively maintained. I am cc’ing Karthik, who may know better update-to-date context and plan for this project. Having said that, if you would like to become a maintainer of this project, it is more than welcomed and we could add your GitHub account to the repository. Please let me know,

Best regards,

yong

On Tue, Nov 2, 2021 at 8:33 AM Furkan Türkal ***@***.******@***.***>> wrote:

TL;DR: I'm wondering if this project is still supported and maintained.

________________________________

We recently used k-bench to run some benchmark test against kubernetes clusters, but we are now a little worried that the project might be unmaintained since no PRs have been merged or issue discussion for a long time. And there has not been a new release. It seems maintainers no longer active on GitHub too: @ganesank-git<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fganesank-git&data=04%7C01%7Cganesank%40vmware.com%7Cab08dc2737974122101408d99e18eb40%7Cb39138ca3cee4b4aa4d6cd83d9dd62f0%7C0%7C0%7C637714652131977236%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=WY50x47f6AeSbpy3FQzKh4TnDVX67mZFlBxq7yTmvlI%3D&reserved=0> @yonglipa<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fyonglipa&data=04%7C01%7Cganesank%40vmware.com%7Cab08dc2737974122101408d99e18eb40%7Cb39138ca3cee4b4aa4d6cd83d9dd62f0%7C0%7C0%7C637714652131987187%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=NRqXbD%2BCBoguk9FfQLwyEc8KKuWKoz%2FfIA8Uf%2FfaeIw%3D&reserved=0> @ganesank<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fganesank&data=04%7C01%7Cganesank%40vmware.com%7Cab08dc2737974122101408d99e18eb40%7Cb39138ca3cee4b4aa4d6cd83d9dd62f0%7C0%7C0%7C637714652131997156%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=NWiqcX1ym1DY91Q%2Bsiycop%2B7MJkZ1nrqJRNR2TlNpvU%3D&reserved=0>

I think looking for new maintainers to help out with the workload would help. This is a pretty cool project which I hope will continue to improve! And we'd like to maintain it!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fvmware-tanzu%2Fk-bench%2Fissues%2F36&data=04%7C01%7Cganesank%40vmware.com%7Cab08dc2737974122101408d99e18eb40%7Cb39138ca3cee4b4aa4d6cd83d9dd62f0%7C0%7C0%7C637714652131997156%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=TTNynuLPdSjSjtrzRMDHfmPS9BtEFjasraZ5C8cILtA%3D&reserved=0>, or unsubscribe<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOW2S7B6HZOLT7TUSAGOBBDUKAAGDANCNFSM5HGYHRUA&data=04%7C01%7Cganesank%40vmware.com%7Cab08dc2737974122101408d99e18eb40%7Cb39138ca3cee4b4aa4d6cd83d9dd62f0%7C0%7C0%7C637714652131997156%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=NbqqIfpX96MLf9RU0KbtdoHs194cQqwZ476KC%2BwM2B8%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cganesank%40vmware.com%7Cab08dc2737974122101408d99e18eb40%7Cb39138ca3cee4b4aa4d6cd83d9dd62f0%7C0%7C0%7C637714652132007118%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ZO8Ko9xCmHKTyakJwlhqQuM4bqcj2IDW5fgFd8U%2Bghk%3D&reserved=0> or Android<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cganesank%40vmware.com%7Cab08dc2737974122101408d99e18eb40%7Cb39138ca3cee4b4aa4d6cd83d9dd62f0%7C0%7C0%7C637714652132007118%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=v7VXn3x4c%2FA7viEBGTsqVJp9jjSK%2BqCmI17%2BWsXigk8%3D&reserved=0>.

**Comment by Dentrax** [comment]
Thank you so much for your prompt response! I do appreciate it! We are looking forward to the new roadmap of the project. If we decide to continue with k-bench then we can always give a hand! Feel free to ping me here when you complete the roadmap. Moreover, you can share the draft here, so we can check this out early and can give some feedback! 🤗
